solana-library/minimum_balance.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+
+// Disclaimer: This library provides functions for working with storage rent. Although it is production ready,
+// it has not been audited for security, so use it at your own risk.
+
+/// Default rental rate in lamports/byte-year.
+///
+/// This calculation is based on:
+/// - 10^9 lamports per SOL
+/// - $1 per SOL
```

Comment:
I don't know too much about Solana. But $1 per SOL seems widely off. Where is this number coming from? Would be good to provide the source

Reply:
I see, thanks